Python/aot_ceval.c

```diff
@@ -5857,7 +6384,11 @@ _PyEval_EvalFrame_AOT_JIT(PyFrameObject *f, PyThreadState * const tstate, PyObje
     f->f_executing = 0;
     tstate->frame = f->f_back;
 
+#ifndef PYSTON_LITE
     return _Py_CheckFunctionResult(NULL, retval, "PyEval_EvalFrameEx");
+#else
+    return retval;
+#endif
```

Comment:
Just curious, why did this have to get added?

Reply:
In 3.9 it got one more argument (tstate) and because we define it as a NOP I thought it's easier to just not call it instead of adding ifdefs for the cases with different number of arguments.